src/Raven.Server/Utils/ExceptionHelper.cs

```diff
@@ -38,6 +38,11 @@ public static bool IsOutOfDiskSpaceException(this Exception ioe)
                                         errorCode == expectedDiskFullError1_Win || errorCode == expectedDiskFullError2_Win;
         }
 
+        public static bool IsMediaWriteProtected(this IOException ioe)
+        {
+            return ioe?.Message.Contains("The media is write protected.", StringComparison.InvariantCulture) ?? false;
```

Comment:
1. Can we drop `.` from the check? It shouldn't matter but it looks a bit strange that we're doing such accuracy. 
2. Is it intentional to use `StringComparison.InvariantCulture` here? Maybe we should ignore casing?

Reply:
1. Yes
2. Dropped